exp/test_cases/socrates_test/socrates_aquaplanet.py

```diff
@@ -94,7 +91,7 @@
         'chunk_size': 16,
         'use_pressure_interp_for_half_levels':False,
         'tidally_locked':False,
-        #'solday': 90
+        'solday':90
```

Comment:
It is noted that this change will change the result of the test case. But this is fine as it will certainly be an obvious change.

Reply:
No action is required on that basis.